hooks/pre/40-el8_modules.rb

```diff
@@ -0,0 +1,7 @@
+if (module_enabled?('katello') || module_enabled?('foreman_proxy_content')) && el8?
+  ensure_dnf_module('pki-core', '10.6')
```

Comment:
I know we care about the postgresql version, but do we care here? Can we use `present`? Should we actually fix this in `puppet-candlepin` instead?

Reply:
Present works for this.